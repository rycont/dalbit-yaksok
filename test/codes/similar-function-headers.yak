약속, 회전하기
    각도 + "도 회전하기" 보여주기

약속, (시간)초 동안 (각도)도 회전하기
    시간 + "초 동안 " + 각도 + "도 회전하기" 보여주기

배열 = [1, 2, 3, 4, 5]
배열[2]초 동안 90도 회전하기