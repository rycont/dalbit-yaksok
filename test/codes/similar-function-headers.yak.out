2초 동안 90도 회전하기
